### ISSUE TITLE
Pin napalm-base driver

### ISSUE THREAD

**Issue by dbarrosop** [napalm-automation/napalm-snmp#8]
See details in the parent issue napalm-automation/napalm-base#107

**Comment by mirceaulinic** [comment]
Let's close this one @dbarrosop

**Comment by dbarrosop** [comment]
I wonder if we should just delete the repo. Haven't seen any activity for a while.

**Comment by mirceaulinic** [comment]
Yes, I think we can delete the repo - if someone would like to contribute at some point we can create it back, although I did not notice massive interest on this.
